### ISSUE TITLE
iOS isLoggedIn always false

### ISSUE THREAD

**Issue by frmrgr** [doomsower/react-native-vkontakte-login#40]
`isLoggedIn` method always resolves with `false` on ios after app restart despite successful authorization. I do not know much about how sdk works, but it [seems](https://github.com/VKCOM/vk-ios-sdk/blob/80d79e4f917de348a24510827c1ba2868fcf7cb5/library/Source/VKSdk.h#L292) that `wakeUpSession` call is required for sdk to restore `accessToken`. At least, I cannot trace (sdk code) how `initializeWithAppId` (which is the only thing called by this library on app launch) can set `vkSdkInstance.accessToken` (which is how `isLoggedIn` works) without calling anything else.

**Comment by doomsower** [comment]
Yeah it looks like Android counterpart does this. I use this module to login into my own backend and then use its credentials, so I didn't have a chance to notice this. 

The difference is that iOS needs permissions scope to wake up, and Android obtains it from current access token (if it exists). Therefore, optional `permissions` argument should be added to `initialize` method of this module. It should be optional to be a non-breaking change, and it's not needed on android anyway. In case if permissions are not provided, iOS bridge should try to get current access token (as android does) and use its permissions to call `wakeUpSession`. 

This shouldn't be difficult, but I don't have time for this at least until Monday. I can review the PR though.

**Comment by frmrgr** [comment]
I think that `permissions` argument is unnecesary, since it is only used to check against already present scope and force logout in case something is missing. Long story short, passing empty array should be just fine if the purpose is to just check whether user is logged. So basically, as I see it, it would be

    [VKSdk wakeUpSession:@[] completeBlock:^(VKAuthorizationState state, NSError *error) {
      if (error)
        reject(error);
      else
        resolve([NSNumber numberWithBool:[VKSdk isLoggedIn]]);
    }

Sorry, I don't wanna do PR since I am not objc programmer.

**Comment by doomsower** [comment]
I just published `react-native-vkontakte-login@0.4.0-beta.2`  
In this version `wakeUpSession` is called during `initialize` on iOS.  
I also added `getAccessToken` method.

Please test it.
